reactjs-jsx-ppx/reactjs_jsx_ppx_v3.ml

```diff
@@ -374,37 +427,25 @@ let makeExternalDecl fnName loc namedArgListWithKeyAndRef namedTypeList =
 
 (* TODO: some line number might still be wrong *)
 let jsxMapper =
-  let jsxVersion = ref None in
```

Comment:
Was this change required for the jsx transform? Or just cleanup?

Reply:
just cleanup!